src/pages/on/[platform].astro

```diff
@@ -0,0 +1,21 @@
+---
+import type { Link } from '../../types.js'
+import site from '../../data/site.js'
+
+interface Props {
+    link: Link
+}
+
+export function getStaticPaths() {
+    return site.socialLinks.map((link) => ({
+        params: { platform: link.name },
+        props: { link }
+    }))
+}
+
+const mastodon = site.socialLinks.find((link) => link.name === 'mastodon')
+const { link } = Astro.props
+---
+
+{mastodon && <link rel="me" href={mastodon.me ?? mastodon.href} />}
```

Comment:
Will the `??` break smoke tests in Node 14?

Reply:
I don't think so because `.astro` is compiled with TypeScript!